specification/chaos/resource-manager/Microsoft.Chaos/preview/2024-12-01-preview/types/faultSimulations.json

```diff
@@ -0,0 +1,150 @@
+{
+    "swagger": "2.0",
+    "info": {
+      "version": "1.0",
+      "title": "Common types used by Resource Providers to implement standard fault simulation actions on resource-types"
+    },
+    "paths": {},
+    "definitions": {
+      "FaultKind": {
+        "type": "string",
+        "description": "The fault scenario to simulate",
+        "enum": [
+          "ZoneDown"
+        ],
+        "x-ms-enum": {
+          "name": "FaultKind",
+          "modelAsString": true,
+          "values": "ZoneDown",
+          "description": "Simulates an availability zone outage on the provider managed resources underlying this resource"
+        }
+      },
+      "FaultSimulationStatus": {
+        "type": "string",
+        "description": "The status of a fault simulation job",
+        "enum": [
+          "Accepted",
+          "Starting",
+          "Active",
+          "Stopping",
+          "Completed",
+          "FailedToStart",
+          "FailedToStop"
+        ]
+      },
+      "FaultSimulation": {
+        "type": "object",
+        "properties": {
+          "simulationId": {
+            "type": "string",
+            "description": "The provider specific ID used to identify and track this simulation"
+          },
+          "status": {
+            "$ref": "#/definitions/FaultSimulationStatus",
+            "description": "Status of this fault simulation"
+          },
+          "submitTime": {
+            "type": "string",
+            "format": "date-time",
+            "description": "The time this fault simulation request was accepted"
+          },
+          "startTime": {
+            "type": "string",
+            "format": "date-time",
+            "description": "The time this fault simulation was started"
+          },
+          "endTime": {
+            "type": "string",
+            "format": "date-time",
+            "description": "The time this fault simulation completed"
+          },
+          "expirationTime": {
+            "type": "string",
+            "format": "date-time",
+            "description": "The absolute expiration time after which this fault should be abandoned or cancelled"
+          },
+          "parameters": {
```

Comment:
if we are going to extend this class, this property is going to be specific to each team, so we can probably remove it or can we define it as an generic object? also a Beter name can be details